config/routes.rb

```diff
@@ -118,7 +118,7 @@
     end
   end
 if Rails.env.production?  
-  match '/404', to: 'errors#file_not_found', via: :all
+  match '/404', to: 'errors#not_found', via: :all
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.